pkg/antctl/transform/networkpolicy/transform.go

```diff
@@ -17,15 +17,26 @@ package networkpolicy
 import (
 	"io"
 	"reflect"
-	"sort"
 	"strconv"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubectl/pkg/cmd/get"
+	"k8s.io/kubectl/pkg/scheme"
+
 	"antrea.io/antrea/pkg/antctl/transform"
 	"antrea.io/antrea/pkg/antctl/transform/common"
 	cpv1beta "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
-	"antrea.io/antrea/pkg/controller/networkpolicy"
 )
 
+var flags = map[string]string{
+	"effectivePriority": "{.tierPriority}",
```

Comment:
I don't think this is correct: effective priority is not the same as `.tierPriority`. The implementation of `func (nps *NPSorter) Less` below, which you are deleting as part of this PR, shows the correct implementation of effective priority.

Reply:
yes for this effectivePriority we need to add a special case, but with the existing implementation I have doubt, that the TierPriority will never be nil, because by default all the TierPriorities are assigned the value of DefaultTierPriority while the network policies are processed. So only extra thing we need to check is that if both the TierPriorities are equal then we need to sort it with respect to Priority.